sys/arm64/arm64/elf_machdep.c

```diff
@@ -583,17 +583,15 @@ elf_reloc_internal(linker_file_t lf, char *relocbase, const void *data,
 		error = LINKER_SYMIDX_CAPABILITY(lf, symidx, 1, &cap);
 		if (error != 0)
 			return (-1);
-		cap += addend;
+		if (addend != 0)
```

Comment:
Yeah this should really be an argument to the method like we do in rtld, but this is fine for now I suppose

Reply:
Could you please add an `/* XXX: ... */` about this being in the wrong place and not breaking sentries?